### PR TITLE
Add keep awake flag handling

### PR DESCRIPTION
**Pull request by andrecardoso**

### CODE REVIEW

File: test/.eslintrc.json [hunk 1]
```diff
@@ -6,5 +6,8 @@
   },
   "parserOptions": {
     "sourceType": "module"
+  },
+  "rules": {
+    "no-magic-numbers": "off"
```
Comment (stulees) [inline test/.eslintrc.json]: @andrecardoso So far we haven't typically turned `no-magic-numbers` off in our other code base. For test coverage you could disable on a per file basis. I think it's worthwhile abiding by the rule in our source.

Comment (santiagonoguez) [reply test/.eslintrc.json]: I agree with Stu for the no-magic-numbers particular case, but other directives may be useful here when needed as this one in my own PR:

https://github.com/Rise-Vision/messaging-service/pull/34/files#diff-1846b85e9e313f50dba02d1d9666c8bc

Comment (andrecardoso) [reply test/.eslintrc.json]: @santiagonoguez @stulees changed. Please review again. Thanks!

**Comment by andrecardoso** [comment]
[(1) Display going into Standby](https://trello.com/c/f2BWl9XS/3982-1-display-going-into-standby)

**Pull request merged**